doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,167 @@
+
+You can decompose a set of relations which is joined by `^` (or) or `&` (and)
+into individual relations using relational atoms. Using {any}`canonical
+<sympy.core.relational.Relational.canonical>` will put order each relation so
+the symbol is on the left, so you can take the right-hand side {any}`rhs
+<sympy.core.relational.Relational.lhs>` to extract the constants:
+
+```py
+>>> from sympy import symbols, solve, Integer, pi
+>>> from sympy.core.relational import Relational
+>>> x = symbols('x')
+>>> one_third = Integer(1)/Integer(3)
+>>> eq = solve([x >= one_third, x**2 <= pi], x)
+>>> eq
+(1/3 <= x) & (x <= sqrt(pi))
+>>> relations = [(i.lhs, i.rel_op, i.rhs) for i in [i.canonical for i in eq.atoms(Relational)]]
+>>> # Sorting relations just to ensure consistent list order for docstring testing
+>>> relations_sorted = sorted(relations, key=lambda x: float(x[2]))
+>>> print(relations_sorted)
+[(x, '>=', 1/3), (x, '<=', sqrt(pi))]
+```
+
+### Extract a tuple of relations
+
+The {any}`args <sympy.core.basic.Basic.args>` of a solution set are the
+individual relations, so you can extract the constants from the left- or
+right-hand side of the `args`:
+
+```py
+>>> from sympy import symbols, solve, Integer, pi
+>>> x = symbols('x')
+>>> eq = solve([x >= Integer(1)/Integer(3), x**2 <= pi], x)
+>>> eq.args
+(1/3 <= x, x <= sqrt(pi))
+>>> constants = []
+>>> for arg in eq.args:
+...     if arg.lhs == x:
+...         constants.append(arg.rhs)
+...     else:
+...         constants.append(arg.lhs)
+>>> constants
+[1/3, sqrt(pi)]
+```
+
+## *Tradeoffs (speed vs. accuracy, etc.) for function*
+
+### *Tradeoff 1*
+
+*Speed-up option 1 content*
+
+### *Speed-up option 2*
+
+*Speed-up option 2 content*
+
+## Not all systems of inequalities can be solved
+
+### Systems of inequalities with no solution
+
+If the system of inequalities has incompatible conditions, for example $x < 0$
+and $x > \pi$, SymPy will return `False`:
+
+```py
+>>> from sympy import symbols, solve, pi
+>>> x = symbols('x')
+>>> solve([x < 0, x > pi])
+False
+```
+
+### Equations with no analytical solution
+
+SymPy may reflect that your equation has no solutions that can be expressed
+algebraically (symbolically) by returning an error such as
+`NotImplementedError`:
+
+```py
+>>> from sympy import symbols, solve, cos
+>>> x = symbols('x')
+>>> solve([cos(x) - x > 0, x > 0], x)
+Traceback (most recent call last):
+    ...
+NotImplementedError: The inequality, -x + cos(x) > 0, cannot be solved using solve_univariate_inequality.
+```
+
+so you may have to solve your equation numerically instead using this [SciPy
+linear programming
+function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.linprog.html).
+
+### Equations which have an analytical solution, and SymPy cannot solve
+
+SymPy has implemented algorithms to solve inequalities involving only one symbol
+(variable), so it cannot solve a set of inequalities involving more than one
```

Comment:
It might be better to indicate that it can solve for a single symbol; this might be removed from `solve` but will always be available through `reduce_inequalities`:
```python
>>> solve([x + y > 1, y > 0], x)
(0 < y) & (y < oo) & (x > 1 - y)
>>> reduce_inequalities([x + y > 1, y > 0], x)
(0 < y) & (y < oo) & (x > 1 - y)
```
But the final relationship, `x < 1` is not currently resolved.

Reply:
Thanks. I'm having trouble understanding the difference between solving for just `x` in the above and solving for `x` and `y`--what would be the result of solving for x and y?

Also, do solve and reduce_inequalities work for only linear inequalities? I ask because I get this error if I include a symbol to the second power (or third or fourth):

```
>>>solve([x ** 2 < y, x > 0], x)
NotImplementedError: 
The inequality, -_y + x**2 < 0, cannot be solved using
solve_univariate_inequality.
```

Reply:
`(0 < y) & (y < oo) & (x > 1 - y)` implies that `y` is in `(0, oo)`, so `x > 1 - y` would mean `x` is in `(-oo, 1)`, so a more complete solution would be `(x < 1) & (y > 0)`.

Also setting `finite=True` would remove the infinity inequalities, which might be worth mentioning in this guide.

Reply:
Thanks. So SymPy can currently reduce a system of inequalities _containing_ more than one symbol, but cannot _reduce for_ more than one symbol. (The phrase "solve for" seems more natural, but we're trying to avoid calling this solving. Although the functions themselves are within `sympy.solvers` and return errors mentioning e.g. "solved"...)

Reply:
I'm not including `finite=True` because it seems to be inapplicable for `reduce_inequalities`.